code/style.css

```diff
@@ -138,3 +138,13 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+
+.VideoBot {
+  width: 20px;
+}
+
+.gifhey{
+  width: 40%;
+  margin-left: -30px;
+}
```

Comment:
Great that you found a way to style the gif! I suggest getting rid of the wrapping `div` and just style the `iframe` like this (give it a class name to not override the other iframes:
```
.gif {
  max-width: 200px;
}
```